server/src/main/resources/configuration.yml

```diff
@@ -43,3 +44,4 @@ httpServerOptions:
   alpnVersions: [ "HTTP_1_1", "HTTP_2" ]
   decompressionSupported: false
   useAlpn: true
+  tracingPolicy: "ALWAYS"
```

Comment:
where is this tracing being used?

Reply:
This is for HTTP tracing in Vertx http server. This will force there is trace created for every http call (including produce).  If there is an existing trace, it will be used instead of creating new one.